meta-client/src/main/java/org/apache/helix/metaclient/factories/MetaClientConfig.java

```diff
@@ -19,23 +19,32 @@
  * under the License.
  */
 
-class MetaClientConfig {
+import org.apache.helix.metaclient.constants.MetaClientConstants;
+
+public class MetaClientConfig {
 
   public enum StoreType {
     ZOOKEEPER, ETCD, CUSTOMIZED
   }
 
   private final String _connectionAddress;
-  private final long _connectionTimeout;
+
```

Comment:
Neat: would it be a good idea to add all "connection/session" related configs in separate class called ConnectionConfig, that way we would just need to pass ConnectionConfig in builder class and in future we can add more params related to connection/session without changing much code.